src/client/components/Navigation/Topnav.js

```diff
@@ -30,14 +32,62 @@ class Topnav extends React.Component {
     location: PropTypes.shape().isRequired,
     history: PropTypes.shape().isRequired,
     username: PropTypes.string,
+    notifications: PropTypes.arrayOf(PropTypes.shape()),
     searchAutoComplete: PropTypes.func.isRequired,
     onMenuItemClick: PropTypes.func,
+    onNotificationClick: PropTypes.func,
+    onSeeAllClick: PropTypes.func,
   };
 
   static defaultProps = {
     autoCompleteSearchResults: [],
+    notifications: [
```

Comment:
Why we need this data here? Can we remove it?

Reply:
yeah was test date